src/java.base/share/classes/java/util/concurrent/atomic/Striped64.java

```diff
@@ -188,24 +190,18 @@ final boolean casCellsBusy() {
     }
 
     /**
-     * Returns the probe value for the current thread.
-     * Duplicated from ThreadLocalRandom because of packaging restrictions.
+     * Returns the ThreadLocalRandom probe value for the current thread.
```

Comment:
I suppose this shouldn't say "current thread" anymore, and advanceProbe uses "the given thread" then it really means the current carrier thread.

Reply:
Thanks. Added "carrier" here and in advanceProbe